tutorphilu_extensions/plugin.py

```diff
@@ -189,12 +189,22 @@
             "openedx-development-settings",
             """
 SUPPORT_URL_DROPDOWN = 'https://edx.readthedocs.io/projects/OPEN-edx-learner-guide/en/open-release-quince.master/SFD_dashboard_profile_SectionHead.html'
+MFE_CONFIG_OVERRIDES = {
+    'course-authoring': {
+        'LOGO_URL': '{% if ENABLE_HTTPS %}https{% else %}http{% endif %}://{{ LMS_HOST }}/static/rg-theme/images/logo-dark.svg'
+    }
+}
 """
         ),
         (
             "openedx-production-settings",
             """
 SUPPORT_URL_DROPDOWN = 'https://edx.readthedocs.io/projects/OPEN-edx-learner-guide/en/open-release-quince.master/SFD_dashboard_profile_SectionHead.html'
+MFE_CONFIG_OVERRIDES = {
+    'course-authoring': {
+        'LOGO_URL': '{% if ENABLE_HTTPS %}https{% else %}http{% endif %}://{{ LMS_HOST }}/static/rg-theme/images/logo-dark.svg'
+    }
+}
 """
```

Comment:
I wonder if modifying the common settings would be a better solution here (instead of duplicating for both dev and prod)

Reply:
@cmltaWt0 what do you think?

Reply:
Do you expect to have a versioning for images? Of not - common settings makes more sense.

Reply:
Decided to go with common settings

Waiting for the local testing now